spec/helpers/application_helper_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe ApplicationHelper, type: :helper do
+
+  context 'methods' do
+    it 'have the date_form_helper method?' do
+      expect(helper).to respond_to(:date_form_helper)
+    end
+
+    it 'have the current_user_id method?' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.